pkg/manifests/manifests.go

```diff
@@ -39,18 +40,17 @@ type Manifests struct {
 	DS   appsv1.DaemonSet
 	Role rbacv1.Role
 	RB   rbacv1.RoleBinding
-	SSC  securityv1.SecurityContextConstraints
+	SCC  securityv1.SecurityContextConstraints
 }
 
-func Get() (*Manifests, error) {
+func Get(ns string) (*Manifests, error) {
```

Comment:
what if `ns == ""`? should it be an error?

Reply:
I the objects would get created on default ns

Reply:
fair, maybe make sure you have test coverage to document/enforce this

Reply:
Done. thanks